src/components/WalletInfo.vue

```diff
@@ -22,38 +25,27 @@
   </div>
 </template>
 
-<script>
-import { defineComponent, toRefs } from 'vue'
+<script setup>
+import { toRefs } from 'vue'
 import { aeWallet } from '../utils/aeternity'
 
-export default defineComponent({
-  name: 'WalletInfo',
-  setup () {
-    const { address, balance, walletStatus, activeWallet } = toRefs(aeWallet)
-
-    return { activeWallet, address, balance, walletStatus }
-  }
-})
+// probably needs deps upgrading deps
+// eslint-disable-next-line
```

Comment:
this acts kinda strange, hopefully will be resolved with https://github.com/aeternity/aepp-boilerplate-vue/issues/13